.evergreen/run-fuzz.sh

```diff
@@ -63,7 +63,7 @@ done
 # If the fuzz directory exists, then tar it up in preparation to upload to S3.
 if [ -d $PROJECT_DIRECTORY/fuzz ]; then
 	echo "Tarring up fuzz directory"
-	tar -czf $PROJECT_DIRECTORY/fuzz.tgz $PROJECT_DIRECTORY/fuzz
+	tar -czf $PROJECT_DIRECTORY/fuzz.tgz --absolute-names $PROJECT_DIRECTORY/fuzz
```

Comment:
Note the fuzz test still fails in CI, but because of a legitimate issue.

Reply:
I think the proper way is to instead `cd $PROJECT_DIRECTORY` and then run `tar -cfz fuzz.tgz ./fuzz`